staging/src/k8s.io/code-generator/examples/apiserver/apis/example/v1/zz_generated.conversion.go

```diff
@@ -140,6 +160,148 @@ func Convert_example_Conversion_To_v1_Conversion(in *example.Conversion, out *Co
 	return autoConvert_example_Conversion_To_v1_Conversion(in, out, s)
 }
 
+func autoConvert_v1_ConversionCustom_To_example_ConversionCustom(in *ConversionCustom, out *example.ConversionCustom, s conversion.Scope) error {
+	out.PublicField = in.PublicField
+	// WARNING: out.privateField is not exported and cannot be set
+	return nil
+}
+
+func autoConvert_example_ConversionCustom_To_v1_ConversionCustom(in *example.ConversionCustom, out *ConversionCustom, s conversion.Scope) error {
+	out.PublicField = in.PublicField
+	// WARNING: in.privateField is not exported and cannot be read
+	return nil
+}
+
+func autoConvert_v1_ConversionCustomContainer_To_example_ConversionCustomContainer(in *ConversionCustomContainer, out *example.ConversionCustomContainer, s conversion.Scope) error {
+	if in.Slice != nil {
+		in, out := &in.Slice, &out.Slice
+		*out = make([]example.ConversionCustom, len(*in))
+		for i := range *in {
+			if err := Convert_v1_ConversionCustom_To_example_ConversionCustom(&(*in)[i], &(*out)[i], s); err != nil {
+				return err
+			}
+		}
+	} else {
+		out.Slice = nil
+	}
+	if in.SliceP != nil {
+		in, out := &in.SliceP, &out.SliceP
+		*out = make([]*example.ConversionCustom, len(*in))
+		for i := range *in {
+			// FIXME: Provide conversion function to convert *ConversionCustom to *example.ConversionCustom
+			compileErrorOnMissingConversion()
```

Comment:
These demonstrate the inability of the generated conversion to find the existing custom conversion function it should call here. The following commit fixes this.